src/compiler/compile/utils/contenteditable.ts

```diff
@@ -0,0 +1,60 @@
+// Utilities for managing contenteditable nodes 
+import Attribute from '../nodes/Attribute';
+import Element from '../nodes/Element';
+
+
+export const CONTENTEDITABLE_ATTR = 'contenteditable';
```

Comment:
Probably a bit overkill to make the attribute name a const export.

Reply:
Yeah generally I would agree with you, but I did this to avoid mis-typing it as `'contentEditable'`, which I did a few times while implementing this PR 😅